plenum/server/replica.py

```diff
@@ -441,6 +441,13 @@ def on_view_change_done(self):
         assert self.isMaster
         self.last_prepared_before_view_change = None
 
+    def on_propagate_primary_done(self):
```

Comment:
The restarted primary will not be able to send checkpoints since it would have lost `checkpoints`